Corona-Warn-App/src/main/res/layout/home_statistics_cards_infections_layout.xml

```diff
@@ -11,15 +32,110 @@
         <TextView
             android:id="@+id/title"
             style="@style/headline5"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/spacing_normal"
+            android:layout_marginTop="@dimen/spacing_normal"
+            android:layout_marginEnd="0dp"
+            android:text="@string/statistics_card_infections_title"
+            app:layout_constraintEnd_toStartOf="@id/statistics_card_info_button"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <androidx.appcompat.widget.AppCompatImageView
+            android:id="@+id/background_image"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:focusable="false"
-            android:layout_marginStart="24dp"
-            android:layout_marginTop="24dp"
-            android:text="Infections"
+            android:layout_marginTop="@dimen/spacing_tiny"
+            android:importantForAccessibility="no"
+            android:src="@drawable/ic_main_illustration_infection"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="@id/layoutFlow" />
+
+        <androidx.constraintlayout.helper.widget.Flow
```

Comment:
Why is this needed?

Reply:
To simplify the layout. Flow widget is the way to go when we want to implement a kinda linear look in ConstraintLayout.